applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -58,27 +55,6 @@ static int cs47l63_comm_reg_conf_write(const uint32_t config[][2], uint32_t num_
 	return 0;
 }
 
-/**@brief Select the on-board HW codec
- */
-static int hw_codec_on_board_set(void)
-{
-	int ret;
-
-	if (!device_is_ready(hw_codec_sel.port)) {
-		return -ENXIO;
-	}
-
-	ret = gpio_pin_configure_dt(&hw_codec_sel, GPIO_OUTPUT_LOW);
-	if (ret) {
-		return ret;
-	}
-
-	/* Allow for switches to flip when selecting on board hw_codec */
-	k_msleep(HW_CODEC_SELECT_DELAY_MS);
```

Comment:
This is no longer needed?

Reply:
Yes, this is now done much earlier by Zephyr. Check out `gpio-hogs`